sc2reader/factories/plugins/replay.py

```diff
@@ -100,21 +100,30 @@ def APMTracker(replay):
     above actions divided by the number of seconds played by the player (not
     necessarily the whole game) multiplied by 60.
     """
+
+    speed_multiplier = 1
+    if replay.expansion == 'LotV':
+        speed_multiplier = 1.4
+
+    game_seconds_per_second = 1.4
+    if replay.expansion == 'LotV':
+        game_seconds_per_second = 1.4
```

Comment:
However, I did find the replay from the test in ggtracker.com and there the avg APM is 40 for the player, so the number 56 for LotV does make sense since 56==40*1.4. Might be a long shot though.

Replay is here (23 minute game):
http://ggtracker.com/players/1668928/Zenchii